### ISSUE TITLE
Accessibility: NVDA is announcing an unnecessary content contain as 'row 0 row 1 columntext column 1' when NVDA focus is on first column row 0

### ISSUE THREAD

**Issue by John-Qiao** [dotnet/winforms#3299]
* .NET Core Version: 
5.0.100-preview.6.20266.3

* Have you experienced this same bug with .NET Framework?: 
Yes

**More info:**

1. This issue can still reproduce in .NET Core 3.1.
2. This issue does not occur in Narrator.
![Narrator](https://user-images.githubusercontent.com/45864985/82185640-1e050180-991c-11ea-8c70-e6f7c877f513.png)

**Problem description:**
NVDA is announcing an unnecessary content contain as 'row 0 row 1 _columntext_ column 1'.
![NVDA](https://user-images.githubusercontent.com/45864985/82185594-09286e00-991c-11ea-893d-39f912c22237.png)

**Expected behavior:**
NVDA should not announce the 'row 0 row 1 _columntext_ column 1' message. 

**Minimal repro:**

1. Launch the attached project in VS and run it.
2. Start NVDA Speech Viewer.
3. Select the cell in DGV and observe the announce in NVDA Speech Viewer.
[TestApp.zip](https://github.com/dotnet/winforms/files/4642640/TestApp.zip)

**Comment by Ryuugamine** [comment]
I have investigated this issue and as I found out we can't control what NVDA will read after selecting an item on the screen. So, we trigger events and NVDA reads the information as defined in the NVDA configuration, Therefore, I think this is an NVDA issue.

**Comment by merriemcgaw** [comment]
There is a rule that states if one of the main screen readers (Narrator in this case) presents the content appropriately then we are compliant. In this case we are presenting the right content but it appears that Narrator is expecting older behavior or just reading excess information. 

@guybark to confirm my conclusion. If @guybark says we're good here we can close the issue.

**Comment by Jenna-Zheng** [comment]
Verified this issue with .Net Core SDK 5.0.100-rc.1.20366, this issue also reproduce and NVDA or Narrator reads  incorrect row value as row 0 for the first row.It should announce row1.
NVDA:
![image](https://user-images.githubusercontent.com/64949616/87757612-83eb0800-c83d-11ea-80ca-6ccfc41d5d7e.png)

Narrator:
![image](https://user-images.githubusercontent.com/64949616/87758505-26f05180-c83f-11ea-9b42-d05f6e12efaf.png)

**Comment by Jenna-Zheng** [comment]
[A11y_.NETCoreWinForms_DataControls_Lower section_Screenreader: Screenreader reads incorrect row value as row 0 for the first row present in the table #1155467 ](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1155467)

**Comment by merriemcgaw** [comment]
Well darn. Thanks for the repro @Jenna-Zheng. @guybark can ignore my tag 🙂

**Comment by Olina-Zhang** [comment]
@merriemcgaw 
Let's summarize screen reader issues for DGV cell:
1. NVDA itself issue: unnecessary content when NVDA focus on DGV cells. This issue cannot repro in Narrator. It needs to confirm if we can control it. 
![image](https://user-images.githubusercontent.com/26474449/89367001-948af180-d70a-11ea-837d-38c48dd2daf0.png)
2. From 'Guy Barker' suggestion "I still would recommend considering **removing the embedded row-related text from the name of the elements**. Now that the Grid/GridItem/Table/TableItem patterns are supported, I’d expect all popular screen readers to be able to access that data and make appropriate announcements. So that would avoid potential duplication of row data in the announcements."
![image](https://user-images.githubusercontent.com/26474449/89367428-8db0ae80-d70b-11ea-9246-b6750c4b3a7f.png)
![image](https://user-images.githubusercontent.com/26474449/89375779-fc4c3700-d720-11ea-8223-4fc7b9a49191.png)

**Comment by Ryuugamine** [comment]
In [original work item](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/956617) from DevDiv we can found the thread with investigation of this issue. You can see that in discussion Guy Barker (CELA) described the essence of the issue in great detail and as a result we can say that this is really an NVDA bug.
